lib/pure/times.nim

```
@@ -932,7 +1002,12 @@ proc `$`*(timeInfo: TimeInfo): string {.tags: [], raises: [], benign.} =
 proc `$`*(time: Time): string {.tags: [TimeEffect], raises: [], benign.} =
   ## converts a `Time` value to a string representation. It will use the local
   ## time zone and use the format ``yyyy-MM-dd'T'HH-mm-sszzz``.
-  $getLocalTime(time)
+  when defined(windows):
+    try: result = $getLocalTime(time)
+    except ValueError:
+      result = $TimeImpl(time) & " seconds before Jan 1st 1970"
```

Perhaps it would be better to just raise an error.